net/xtables-addons/Makefile

```diff
@@ -61,8 +61,7 @@ endef
 # 2: extension/module display name used in package title/description
 # 3: list of extensions to package
 # 4: list of modules to package
-# 5: module load priority
-# 6: module depends
+# 5: module depends
```

Comment:
This is removed why? How is it relevant to adding kmod-lib-textsearch dependency?

Reply:
I corrected incorrect numbering in the description and didn't mention it. I'll fix it.

Reply:
Are you sure that it is indeed incorrect numbering? 🤔 You removed load priority at all.

https://github.com/openwrt/packages/pull/26368/files#diff-883afbcc5b201d77cf79e81dcaa0e668ad9d742d963e6fa5cde57fd8b9eef6adR197

Reply:
There are 5 arguments in the calls.

Reply:
It was removed in https://github.com/openwrt/openwrt/commit/f12f4074aff6536a8056689406197809be373685#diff-a15b082a2215a916025d04f919074e5b4de8ac49a861998a41af060368cac1a9 , but your change is still only partial as I pointed already and of course, it should be included in the separate commit not in commit which adds dependency.